### PR TITLE
Feature/add days per week option

### PR DESCRIPTION
**Pull request by sepiott**
Adds the ability to choose a type of week, between the options of a standard 7 day week or a 5 day working week. 

There are also two fixes for the language switcher included: 
- The Dropdown wasn't shown, when there was no language saved in the local storage due to an javascript undefined variable error. 
- The Aria Parameter "activedescendant" and the list items id wasn't dynamicly set and therefor was the same for every option and id

### CODE REVIEW

**Comment by lostdesign** [comment]
Thanks for your PR Seb, is it okay if I add electron-store beforehand in the next release as I really don't want to use the localstorage anymore? The recent language dropdown bug could have been prevented by a migration for example :D 

I am planning on adding the electron-store within the next two days. Could you then move the localstorage property into the electon-store option?

**Comment by lostdesign** [comment]
Noticed a bug where using the keyboard or menu items to skip a single day would select saturday and sunday as it skips one day at a time. 

Could you implement to skip sat/sun when working week is selected? In both directions, so if monday and you go back, skip to friday, if friday, skip to monday.

File: src/components/day-switcher.vue [hunk 1]
```diff
@@ -3,8 +3,8 @@
     class="
       flex
       dark:bg-black
-      justify-center
-      space-x-4
+      justify-between
```
Comment (lostdesign) [inline src/components/day-switcher.vue:6]: can we keep that justify-center, as i removed the width restriction and having the days loose would not really work well

Bildschirmfoto 2021-08-10 um 08 29 58

File: src/components/day-switcher.vue [hunk 2]
```diff
@@ -3,8 +3,8 @@
...
+      px-10
```
Comment (lostdesign) [inline src/components/day-switcher.vue:7]: guess this might not be necessary then as well

File: src/components/days-per-week-dropdown.vue [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+import TickIcon from '@/assets/icons/tick.svg'
+import DropdownIcon from '@/assets/icons/dropdown.svg'
+import { mapActions } from 'vuex'
+import { Actions as CalendarActions } from '@/store/modules/calendar/types'
+
+export default {
+  data() {
+    return {
+      open: false,
+      selected: parseInt(localStorage.daysPerWeek) || 7,
```
Comment (lostdesign) [inline src/components/days-per-week-dropdown.vue:64]: You may now use `AppGetters.GET_DAYS_PER_WEEK` from vuex and need to store the value inside `electron-store`. You can take a look at how the theme is made, all necessary code is in the `<theme-switcher>` and `vuex/modules/app/..`

File: src/components/days-per-week-dropdown.vue [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  },
+  methods: {
+    ...mapActions('calendar', [CalendarActions.SET_CURRENT_WEEK]),
+    _handleDaysPerWeekOptionsChange(option) {
+      if (localStorage.daysPerWeek === option) return
+
+      this.selected = option
+      this.open = false
+      localStorage.daysPerWeek = option
+      this.setCurrentWeek()
```
Comment (lostdesign) [inline src/components/days-per-week-dropdown.vue:76]: This only was necessary for the language changes, I guess you could remove it and it should work fine? I needed to rerender the day selectors as they are INTL based

**Review by lostdesign — CHANGES_REQUESTED** [review]
The electron-store had been pushed, you may now use that.

File: src/components/days-per-week-dropdown.vue (hunk 4 above)
Comment (sepiott) [reply src/components/days-per-week-dropdown.vue:76]: I tried it, but without this, the day-switcher will not chance, when the setting is changed

Comment (lostdesign) [reply src/components/days-per-week-dropdown.vue:76]: Ohh damn, you are right lol. It would recalculate the week. Without it would not mutate the week.

**Review by lostdesign — CHANGES_REQUESTED** [review]
Changing days didn't work per shortcut, as `this.isWorkWeek()` was called as function while it is a computed property.

**Pull request closed without merging**